### DISCUSSION TITLE
Upload fails althoug max. filesize is increased accordingly

### DISCUSSION THREAD

**Discussion by michael-dietl** [pocketbase/pocketbase#2960] [Q&A]
I have implemented a collection to upload *.mp4 files. As they may be very large I have increased max. filesize accordingly

![image](https://github.com/pocketbase/pocketbase/assets/9390335/85ca77b6-4ba8-46df-be2b-92cb5d5f9390)

Unfortunately there is a http error 413

![image](https://github.com/pocketbase/pocketbase/assets/9390335/4f3df6d6-ccc8-40f9-b101-18a557192718)

As far as I remember there is no limitation on HTTP Post Requests.

:-)

**Comment by ganigeorgiev** [comment]
I don't think the 413 error is from PocketBase as we don't throw it explicitly.

Do you have a reverse proxy (nginx, apache, caddy, etc.)? If Yes, make sure to check the allowed body size limit (eg. for nginx [the default `client_max_body_size` is 1mb](https://nginx.org/en/docs/http/ngx_http_core_module.html#client_max_body_size)).

**Comment by michael-dietl** [comment]
Yes, you are right. Pocket Base runs behind a Apache. Sorry - I will check myself

**Comment by michael-dietl** [comment]
I have solved my problem.

First mod_security prevented the upload by one rule and secondly I had to increase the timeout at the proxy rule

`ProxyPass / http://127.0.0.1:8090/ timeout=600`

Maybe it helps someone.
